### PR TITLE
Update ES_es.json

### PR DESCRIPTION
**Pull request by CarlosNunezMX**
Updated Spanish locations after website changes.

### CODE REVIEW

**Comment by CarlosNunezMX** [comment]
Any review? New or something?...

File: locales/ES_es.json [hunk 1]
```diff
@@ -177,34 +176,27 @@
 				"github": "https://github.com/Kinnay"
 			},
 			{
-				"name": "NinStar",
-				"caption": "Icons for the Mii Editor and Juxt reactions",
-				"picture": "https://github.com/ninstar.png",
-				"github": "https://github.com/ninstar"
-			},
```
Comment (ashmonty) [inline locales/ES_es.json:184]: NinStar is missing in your locale

Comment (ashmonty) [inline locales/ES_es.json:184]: NinStar is missing in your locale

File: locales/ES_es.json [hunk 2]
```diff
@@ -215,30 +207,14 @@
...
-				{
-					"header": "Install Pretendo",
-					"caption": "View the setup instructions"
-				},
-				{
-					"header": "Got an error?",
-					"caption": "Search for it here"
-				}
-			]
-		}
```
Comment (ashmonty) [inline locales/ES_es.json:242]: This part is also missing in your locale

**Review by ashmonty — CHANGES_REQUESTED** [review]
Sorry, only just now had time to review. Some strings seem to be missing in your locale, could you please fix that?

File: locales/ES_es.json (hunk 1 above)
Comment (CarlosNunezMX) [reply locales/ES_es.json:184]: Ready, I will to fix it. Sorry for the inconvenients

**Comment by CarlosNunezMX** [comment]
Please wait a moment, i don't know what i did

**Comment by CarlosNunezMX** [comment]
i dont know. See ya!

**Pull request closed without merging**